modules/ROOT/pages/getting-started.adoc

```diff
@@ -1,142 +1,67 @@
 :experimental:
 = Fedora CoreOS - Getting Started
 
-Fedora CoreOS has no install-time configuration. Every Fedora CoreOS system begins with a generic, unconfigured disk image. On first boot Ignition will read the supplied config and configure the system. Ignition configs are usually supplied via the cloud’s userdata mechanism, or, in the case of bare metal, injected at install time. This guide will show you how to launch Fedora CoreOS on AWS, QEMU, and bare metal as well as how to create Ignition configs.
+== Background
+
+Fedora CoreOS (FCOS) has no install-time configuration. Every FCOS system begins with a generic disk image. For each deployment mechanism (bare metal, local VM, or cloud VM), the configuration must be supplied at first boot. This is done via a configuration file and FCOS Ignition. For cloud deployments, Ignition supplies the configuration via the cloud’s userdata mechanism. In the case of bare metal, Ignition injects the configuration at install time.
```

Comment:
s/FCOS Ignition/Ignition/ (and linking to https://github.com/coreos/ignition would be good).

s/supplies/gathers/ perhaps?

> In the case of bare metal, Ignition configuration can be injected on disk.

Reply:
Thanks for the clarification.